src/Lecture2.hs

```diff
@@ -97,7 +111,8 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+dropSpaces :: [Char] -> [Char]
+dropSpaces string = takeWhile (not . isSpace) $ dropWhile isSpace string
```

Comment:
You can use function composition and eta-reduce the definition here as well
```suggestion
dropSpaces = takeWhile (not . isSpace) . dropWhile isSpace
```

Reply:
you are quite right. I already figured out and did it, just forgot to commit. First time it was tough to understand eta-reduction but right now it is easier.